app/helpers/flood_risk_engine/enrollments/addresses_helper.rb

```diff
@@ -0,0 +1,26 @@
+module FloodRiskEngine
+  module Enrollments
+    module AddressesHelper
+
+      def new_address_path(enrollment:, addressable:, postcode:, address_type:, step_back_to_postcode: false)
+        new_enrollment_address_path(
+          enrollment,
+          addressable_id: addressable.id,
+          addressable_type: addressable.class.to_s,
+          postcode: postcode,
+          address_type: address_type,
+          step_back_to_postcode: step_back_to_postcode
+        )
+      end
+
+      # The change postcode page should either be the current step or the previous step.
+      # set :step_back_to_postcode to true if the postcode step will be the
+      # previous step.
```

Comment:
hard to follow really what this is for, but superficially seems like the kind of thing that the state engine should be enforcing /enabling through events and the allowable transitions between states